easydata.net/src/EasyData.EntityFrameworkCore.Relational/Services/EasyDataManagerEF.cs

```diff
@@ -196,6 +196,45 @@ public override async Task DeleteRecordAsync(string modelId, string sourceId, JO
             await DbContext.SaveChangesAsync(ct);
         }
 
+        /// <inheritdoc />
+        public override async Task DeleteRecordsInBulkAsync(string modelId, string sourceId, JObject props, CancellationToken ct = default)
+        {
+            ct.ThrowIfCancellationRequested();
+            var entityType = GetCurrentEntityType(DbContext, sourceId);
+            var recordsPKs = GetRecordsPKs(props);
+            var recordsToDelete = new List<object>();
+
+            foreach (var pk in recordsPKs) {
+                var keys = GetKeys(entityType, pk);
+                var record = FindRecord(DbContext, entityType.ClrType, keys.Values);
+
+                if (record == null) {
+                    throw new RecordNotFoundException(sourceId,
+                        $"({string.Join(";", keys.Select(kv => $"{kv.Key.Name}: {kv.Value}"))})");
+                }
+
+                recordsToDelete.Add(record);
+            }
+
+            DbContext.RemoveRange(recordsToDelete);
+            await DbContext.SaveChangesAsync(ct);
+        }
+
+        /// <summary>
+        /// Get primary keys of records from the request body.
+        /// </summary>
+        private IEnumerable<JObject> GetRecordsPKs(JObject fields)
+        {
+            foreach (var keyValue in fields) {

```

Comment:
Can we instead specify a model for this case and attempt to deserialize `JObject` to that model type?